src/Microsoft.Diagnostics.Runtime/src/Common/ClrValueClass.cs

```diff
@@ -142,5 +142,8 @@ private ulong GetFieldAddress(string fieldName, ClrElementType element, string t
             ulong address = field.GetAddress(Address, _interior);
             return address;
         }
+
+        public bool Equals(IAddressableTypedEntity other)
+            => Address == other?.Address && Type == other?.Type;
```

Comment:
NitPick:  `=> other != null && Address == other.Address && Type == other.Type`.  This avoids creating a `ulong?` with that comparison.

Reply:
Agree, done.